From 060d935b09ff40c9fddc08928a7b0351e7a41968 Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 14:58:21 -0400
Subject: [infiniband] usnic: Add dependency on CONFIG_INET

Message-id: <4d7f872b2732b80583eecb21491a733b4fc69279.1411569661.git.dledford@redhat.com>
Patchwork-id: 96583
O-Subject: [Patch RHEL7 19/29] IB/usnic: Add dependency on CONFIG_INET
Bugzilla: 916382
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Upinder Malhi <umalhi@cisco.com>

usNIC needs inet notifiers to function correctly, so add a Kconfig
dependency on CONFIG_INET.

Signed-off-by: Upinder Malhi <umalhi@cisco.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 2d97436f5b06217beb6c91a0cd0ae0f0d79b61cc)
Signed-off-by: Doug Ledford <dledford@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/usnic/Kconfig b/drivers/infiniband/hw/usnic/Kconfig
index 2cc8ba0..29ab11c 100644
--- a/drivers/infiniband/hw/usnic/Kconfig
+++ b/drivers/infiniband/hw/usnic/Kconfig
@@ -1,6 +1,6 @@
 config INFINIBAND_USNIC
  tristate "Verbs support for Cisco VIC"
- depends on NETDEVICES && ETHERNET && PCI && INTEL_IOMMU
+ depends on NETDEVICES && ETHERNET && INET && PCI && INTEL_IOMMU
  select ENIC
  select NET_VENDOR_CISCO
  select PCI_IOV
-- 
1.7.1